src/backend/src/utils/google-integration.utils.ts

```diff
@@ -168,3 +172,172 @@ export const downloadImageFile = async (fileId: string) => {
     throw error;
   }
 };
+
+/**
+ * Creates a new google calendar on the NER google calendar
+ * @param name
+ * @returns the calendar id
+ */
+export const createCalendar = async (name: string) => {
+  if (process.env.NODE_ENV !== 'production') return;
+  try {
+    oauth2Client.setCredentials({
+      refresh_token: CALENDAR_REFRESH_TOKEN
+    });
+    const calendar = google.calendar({ version: 'v3', auth: oauth2Client });
+    const createdCalendar = await calendar.calendars.insert({
+      requestBody: { summary: `NER ${name} Meetings`, description: `A Team Type Within NER` }
+    });
```

Comment:
Since we want multiple organizations to use this, this more general. The summary could be like `[name] Meetings` and the description could be `Meetings for the [name] system` or something